codelists/financingArrangementType.csv

```diff
@@ -0,0 +1,8 @@
+Code,Title,Description
+bond,Bond,"A debt instrument issued by a national government, subnational government, project company or corporate entity. A bond typically obliges the issuer to make interest payments (coupons) over a specified period and to repay the principal (amount borrowed) at the maturity date."
+loan,Loan,"A debt instrument provided by a lender such as a commercial bank, multi-lateral financial institution, or development agency. A loan typically obliges the borrower to make principal and interest payments over a specified period."
```

Comment:
I'm not quite sure that 'issued by a lender' is the right terminology for loans, since 'issuer' usually refers to the borrower in debt financing.

Reply:
I think instruments are always issued.

For a loan, it is the lender (bank) that ultimately decides whether to lend (issue a loan) to the borrower.

In the case of a bond, it's reversed: the borrower (govt/company) decides whether to issue a bond.

Reply:
Yes, that was what I thought, but the Grant Thornton summary of IFRS 9 has the issuer and the lender as the two parties in a debt instrument:

> Debt instruments are contractual obligations of the issuer to repay the lender in accordance with a specified maturity and under the contractual terms.

Likewise, the IFRS's [illustrative examples for IFRS 9](https://www.aasb.gov.au/admin/file/content105/c9/IFRS9_IE_7-14.pdf) describes the borrower as an issuer in the context of a loan and uses the term "origination" rather than "issuing" to describe the lender's role:

> Company Y has a funding structure that includes a senior secured loan facility with different tranches. Bank X provides a tranche of that loan facility to Company Y. At the time of origination of the loan by Bank X, although Company Y’s leverage was relatively high compared with other issuers with similar credit risk, it was expected that Company Y would be able to meet the covenants for the life of the instrument.

I am happy for you to make a final call on "provided" vs "issued" vs "originated" - just sharing some of the reasoning about why I had avoided using "issued" in the context of a loan.

Reply:
Let's use "provided" for both bond and loan for consistency. We'll only use "issuer" to refer to the counter-party to the bondholder.

Reply:
Done in https://github.com/open-contracting-extensions/ocds_finance_extension/pull/26/commits/66a97cb90b026638b54321a52fc8d857d3bbe01f